### DISCUSSION TITLE
Community Recommendations for Game Asset Developers

### DISCUSSION THREAD

**Discussion by BenjamenMeyer** [vegastrike/Vega-Strike-Engine-Source#984] [General]
Vega Strike is game engine that will be used to develop several games. In fact there is an outline of 4 games related to (and including) the VS:UtCS game that we are ourselves developing assets for. As we want to encourage more people to use the engine to develop more games it is also necessary to provide guidance on what to use for developing those games.

The below is to start capturing questions that will need to be answered to help people along.

Questions:
- Should game asset developers use master or only stable releases?
- What tools should we recommend?

**Comment by ermo** [comment]
From previous experience porting Privateer: Wing Commander Universe (Privateer: Parallel Universe continuation project) to python3, porting against a stable set of VS:UtCS asset python scripts that worked with a stable VS C++ engine branch was a huge benefit.

Don't know how much that data point matters, but there it is.

**Comment by BenjamenMeyer** [comment]
@ermo we're going to pull some of the python scripts from UtCS over to the engine where it makes sense to use across multiple assets, so hopefully that would help with PWCU too and help with not having to rely on UtCS in the long run. But yes I can certainly see how having a fully working example set that way would be beneficial.

Hopefully we can build out such things as part of a test suite and example suite aside from a specific game.
